packages/graphile-build-pg/src/plugins/PgMutationCreatePlugin.js

```diff
@@ -44,6 +44,11 @@ export default (function PgMutationCreatePlugin(
           .filter(table => !!table.namespace)
           .filter(table => table.isSelectable)
           .filter(table => table.isInsertable)
+          .filter(
+            table =>
+              (pgEnableDefaultMutationTables.length == 0) ||
+              (pgEnableDefaultMutationTables.indexOf(table.name) >= 0)
```

Comment:
`table.name` is not sufficient because tables with the same name might be sourced from multiple schemas; you'll need a bit of a more complex check here that takes the schema into consideration also (maybe a tuple?)

Reply:
How about `schema.table` ? (e.g. public.post)
I think that it is easier to pass from cli (--enable-default-mutation-tables public.post).
If tuple, how should I receive the value of this option?